packages/react-checkbox/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -71,15 +74,21 @@ export type CheckboxSlots = {
 /**
  * Checkbox Props
  */
-export type CheckboxProps = Omit<ComponentProps<CheckboxSlots, 'input'>, 'size' | 'checked' | 'defaultChecked'> &
+export type CheckboxProps = Omit<
+  ComponentProps<CheckboxSlots, 'input'>,
+  'size' | 'checked' | 'defaultChecked' | 'onChange'
```

Comment:
Omitting `'onChange'` isn't really necessary because our version extends the native one.

Reply:
Ignore this comment; I was wrong 😕